keras_cv/backend/__init__.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from keras_cv.backend.config import multi_backend
+
+# Keys are of the form: "module.where.attr.exists->module.where.to.alias"
+# Value are of the form: ["attr1", "attr2", ...] or
+#                        [("attr1_original_name", "attr1_alias_name")]
+_KERAS_CORE_ALIASES = {
```

Comment:
My main question is if we should add these shims forwards or backwards. On KerasNLP I put the shims in for legacy Keras version, so we can always access, e.g. `serialize_keras_object` via `keras.saving.serialize_keras_object`, which is the new path. For tf.keras this will be live in 2.13.

That seems ideal, because our codebase can represent the latest and greatest keras practices, and when we want to drop the legacy support, we just delete it out here.

Reply:
I think this is a fair point. Either way, we should be consistent. So I've inverted these.